src/main/java/com/hubspot/jinjava/lib/fn/Functions.java

```diff
@@ -386,15 +388,21 @@ public static int movePointerToJustBeforeLastWord(int pointer, String s) {
     "With two parameters, the range will start at the first value and increment by 1 up to (but not including) the second value. " +
     "The third parameter specifies the step increment. All values can be negative. Impossible ranges will return an empty list. " +
     "Ranges can generate a maximum of " +
-    RANGE_LIMIT +
-    " values.",
+    DEFAULT_RANGE_LIMIT +
+    " values by default, but this integer value is configurable.",
```

Comment:
it is configurable, but not by the user which is who this documentation is targeted to.
```suggestion
    " values.",
```